### PR TITLE
Moving core components from wstring->string

### PR DESCRIPTION
**Pull request by dimitry-**
This will reduce number of times we need to convert between wide and utf8 strings in our core coponent. The goal is to use wstrings only when invoking windows API that requires them and use regular strings internally.

### CODE REVIEW

File: OrbitCore/ScopeTimer.cpp [hunk 1]
```diff
@@ -6,83 +6,71 @@
...
-    tabs += L"  ";
+LocalScopeTimer::LocalScopeTimer(const std::string& message)
+    : millis_(nullptr), message_(message) {
+  std::string tabs;
+  for (size_t i = 0; i < CurrentDepthLocal; ++i) {
+    tabs += "  ";
   }
-  PRINT(Format(L"%lsStarting %ls...\n", tabs.c_str(), m_Msg.c_str()));
+  PRINT(
+      absl::StrFormat("%lsStarting %ls...\n", tabs.c_str(), message_.c_str()));
```
Comment (dpallotti) [inline OrbitCore/ScopeTimer.cpp]: Shouldn't %ls also be changed to %s?

File: OrbitCore/ScopeTimer.cpp [hunk 2]
```diff
@@ -6,83 +6,71 @@
...
-      tabs += L"  ";
+  if (!message_.empty()) {
+    std::string tabs;
+    for (size_t i = 0; i < CurrentDepthLocal; ++i) {
+      tabs += "  ";
     }
 
-    PRINT(Format(L"%ls%ls took %f ms.\n", tabs.c_str(), m_Msg.c_str(),
-                 m_Timer.ElapsedMillis()));
+    PRINT(absl::StrFormat("%ls%ls took %f ms.\n", tabs.c_str(),
```
Comment (dpallotti) [inline OrbitCore/ScopeTimer.cpp]: Same as above.

File: OrbitCore/Capture.cpp [hunk 3]
```diff
@@ -542,13 +543,14 @@ void Capture::SaveSession(const std::wstring& a_FileName) {
...
-  std::wstring saveFileName = a_FileName;
+  std::string saveFileName = ws2s(a_FileName);
   if (!EndsWith(a_FileName, L".opr")) {
-    saveFileName += L".opr";
+    saveFileName += ".opr";
   }
 
-  SCOPE_TIMER_LOG(Format(L"Saving Orbit session in %s", saveFileName.c_str()));
-  std::ofstream file(ws2s(saveFileName), std::ios::binary);
+  SCOPE_TIMER_LOG(absl::StrFormat("Saving Orbit session in %s",
```
Comment (dpallotti) [inline OrbitCore/Capture.cpp]: LocalScopeTimer used by SCOPE_TIMER_LOG doesn't seem to have a constructor that accepts std::string, only std::wstring.

Comment (dimitry-) [reply OrbitCore/Capture.cpp]: I removed the wstring c-tor and replaced it with std::sting as well - see below.

File: OrbitCore/ScopeTimer.cpp (hunk 1 above)
Comment (dimitry-) [reply OrbitCore/ScopeTimer.cpp]: yep - it should. fixed.

File: OrbitCore/ScopeTimer.cpp (hunk 2 above)
Comment (dimitry-) [reply OrbitCore/ScopeTimer.cpp]: done

**Pull request merged**